test/media/streaming_engine_unit.js

```diff
@@ -2748,6 +2733,41 @@ describe('StreamingEngine', function() {
       expect(onEvent).not.toHaveBeenCalled();
       expect(onManifestUpdate).toHaveBeenCalled();
     });
+
+    it('adds timeline regions for emsg events', function () {
+      videoStream1.containsEmsgBoxes = true;
+      segmentData[ContentType.VIDEO].segments[0] = getEmsgSegment();
+
+      // Here we go!
+      streamingEngine.init();
+      runTest();
+
+      var expectedEmsg = getExpectedEmsg();
```

Comment:
We are now enforcing the Google Style Guide requirement that "var" be replaced with "let" or "const".

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

If you rebase your change, you will see the new linter complain about this.